src/main/java/net/jingles/enchantments/Commands.java

```diff
@@ -27,124 +27,148 @@
   public boolean onCommand(CommandSender sender, Command command, String name, String[] args) {
 
     if (args.length < 1) {
-      sender.sendMessage(ERROR + "Incorrect arguments.");
+      sender.sendMessage(ERROR + "Incorrect arguments: /enchantments help");
```

Comment:
If you repeat a String a lot, just make another static final String field so you can reuse it without having to create a new instance.